docs/src/install/installation.rst

```diff
@@ -5,7 +5,7 @@
 .. _installation:
 
 ********************************************************************
-Installation
+Tensile installation
 ********************************************************************
 
 Install ROCm
```

Comment:
![image](https://github.com/user-attachments/assets/f842ea4c-f12c-42a1-a689-fe62c3155e76)

Reply:
remove "only" above

Reply:
I have replaced it with:
Alternatively, export the path exclusively for your current shell session, using ``export PATH=/opt/rocm/bin/:$PATH``.